rules/falco_rules.yaml

```diff
@@ -1040,13 +1043,16 @@
               and fd.name startswith "/etc/dd-agent")
 
 - macro: rancher_writing_conf
-  condition: (((proc.name=healthcheck and container.image.repository contains "rancher/healthcheck") or
-               (proc.name=lb-controller and container.image.repository contains "rancher/lb-service-haproxy") or
-               (proc.name=rancher-dns and container.image.repository contains "rancher/dns")) and 
+  condition: ((proc.name in (healthcheck, lb-controller, rancher-dns)) and
+              (container.image.repository contains "rancher/healthcheck" or
+               container.image.repository contains "rancher/lb-service-haproxy" or
+               container.image.repository contains "rancher/dns") and
               (fd.name startswith "/etc/haproxy" or fd.name startswith "/etc/rancher-dns"))
 
 - macro: rancher_writing_root
-  condition: (proc.name=rancher-metadat and container.image.repository contains "rancher/metadata" and fd.name startswith "/answers.json")
+  condition: (proc.name=rancher-metadat and
```

Comment:
is this process name right?

Reply:
yes, like @mstemm pointed out in my previous PR, the falco has a limit for the process name which is 15.

Reply:
ok thank you